### PR TITLE
add a short doc about config.json

### PR DESCRIPTION
**Pull request by jakubgs**
During CUD some people complained that we didn't have docs about configuring `status-go`.

For easier reading: [add-config-doc/config](https://github.com/status-im/status-go/tree/add-config-doc/config)

### CODE REVIEW

File: config/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+{
+    "WhisperConfig": {
+        "Enabled": true,
+        "EnableMailServer": true,
+        "LightClient": false,
+        "MailServerPassword": "status-offline-inbox"
+    }
+}
+```
+The `MailServerPassword` is used for symmetric encryption of history requests.
```
Comment (dshulyak) [inline config/README.md:68]: we can mention that this password will be used when mail server is added in status app.
@cammellos maybe you have a link to documentation that explains that?

**Review by dshulyak — APPROVED** [review]

File: config/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+This document describes the available options in the JSON config for `status-go`.
+
+The structure of the JSON config is defined in the [`params/config.go`](/params/config.go) file, which also contains details comments on meaning of each option.
```
Comment (adambabik) [inline config/README.md]: I would add that `NodeConfig` is the main struct one should start looking from.

File: config/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    "LogFile": "/var/log/status-go.log",
+    "LogLevel": "INFO",
+    "LogMaxSize": 200,
+    "LogMaxBackups": 5,
+    "LogCompressRotated": true
+}
+```
+Valid `LogLevel` settings are: `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`
+
+## `LightEthConfig`
```
Comment (adambabik) [inline config/README.md]: I would remove it because we don't support LES.

File: config/README.md (hunk 1 above)
Comment (adambabik) [reply config/README.md:68]: I would add that `status-offline-inbox` is the password used by Status mail servers listed on https://fleets.status.im.

**Review by adambabik — APPROVED** [review]

File: config/README.md (hunk 1 above)
Comment (jakubgs) [reply config/README.md:68]: Good idea.

File: config/README.md (hunk 2 above)
Comment (jakubgs) [reply config/README.md]: Will do.

File: config/README.md (hunk 3 above)
Comment (jakubgs) [reply config/README.md]: Less noise is always nice.

**Pull request merged**